cmd/tapcli/assets.go

```diff
@@ -187,15 +199,26 @@ var finalizeBatchCommand = cli.Command{
 	ShortName:   "f",
 	Usage:       "finalize a batch",
 	Description: "Attempt to finalize a pending batch.",
-	Action:      finalizeBatch,
+	Flags: []cli.Flag{
+		cli.BoolFlag{
+			Name: shortResponseName,
+			Usage: "if true, then the current assets within the " +
+				"batch will not be returned in the response " +
+				"in order to avoid printing a large amount " +
+				"of data in case of large batches",
+		},
+	},
+	Action: finalizeBatch,
 }
 
 func finalizeBatch(ctx *cli.Context) error {
 	ctxc := getContext()
 	client, cleanUp := getMintClient(ctx)
 	defer cleanUp()
 
-	resp, err := client.FinalizeBatch(ctxc, &mintrpc.FinalizeBatchRequest{})
+	resp, err := client.FinalizeBatch(ctxc, &mintrpc.FinalizeBatchRequest{
+		ShortResponse: ctx.Bool(shortResponseName),
```

Comment:
Alternatively, we can just do this on the CLI side? Still getting thru the diff tho.